src/main/scala/SNSHandler.scala

```diff
@@ -1,34 +1,51 @@
-import java.util.Base64
-
 import com.amazonaws.SdkBaseException
-import com.amazonaws.services.sns.AmazonSNSClient
+import com.amazonaws.services.sns.{AmazonSNS, AmazonSNSClientBuilder}
 import com.amazonaws.services.sns.model.PublishRequest
-import com.amazonaws.services.kinesis.model.Record
+import com.gu.crier.model.event.v1.{Event, EventPayload, ItemType, _}
 import org.apache.logging.log4j.scala.Logging
 
-import scala.concurrent.Future
-import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.{ExecutionContext, Future}
+import io.circe._
+import io.circe.generic.auto._
+import io.circe.syntax._
+import com.gu.contentapi.json.CirceEncoders._
```

Comment:
This pulls in all of the Encoders that have already been generated by the content-api-models library, which saves having to generate them again here at compile time (you have to define them in the correct order otherwise it takes longer than the age of the universe to compile).